### ISSUE TITLE
Computations doesn'

### ISSUE THREAD

**Issue by felixcremer** [meggart/YAXArrays.jl#19]

**Comment by meggart** [comment]
I have no idea why the progress bar does not show up, but can confirm that it is broken for me as well. One reason might be that the Progress bar only updates when a full chunk of data is processed, so it will never show for very small arrays. However, it is also broken for me for large arrays. You might have found out that this https://github.com/meggart/YAXArrays.jl/blob/master/src/DAT/DAT.jl#L418-L424 is the portion of code to look at.

**Comment by felixcremer** [comment]
It seems, that the fix for #16 fixed the ProgressBar as well. 

Sometimes the ProgressBar is not shown, because the `length(allRanges)`is one and there is only one computation. I am wondering, whether we should simply print that there is no progress bar in this case, because this would keep the user informed what is happening and that something is happening.

**Comment by meggart** [comment]
Yes, this might be a goo option. Alternatively, we might add another channel in this whole communication progress, which updates the state of the current worker after every N iterations. This would allow for a more fine-grained progress bar, even if everything is happening within a single chunk.

**Comment by meggart** [comment]
We should add a warning that if `length(allRanges)==1` that no progress bar will be shown.
